src/MOI_wrapper.jl

```diff
@@ -43,6 +43,8 @@ const SUPPORTED_VECTOR_SETS = Union{
     MOI.PositiveSemidefiniteConeTriangle,
 }
 
+const SUPPORTED_TERMINATION_STATUS = [MOI.LOCALLY_SOLVED, MOI.OPTIMAL, MOI.ALMOST_OPTIMAL]
```

Comment:
When do we have cases with `MOI.LOCALLY_SOLVED`?

Reply:
yep. i remember differentiating a convex QP in some solver gave this result. let me fetch the history